### PR TITLE
Lear 24 topbar

### PR DESCRIPTION
**Pull request by tozaicevas**
TopBar (not all routes working yet, e.g. subordinates)

### CODE REVIEW

**Comment by tozaicevas** [comment]
This is WIP. I am currently changing whole navigation, redux logic

File: frontend/src/pages/SignIn.js [hunk 1]
```diff
@@ -20,25 +23,28 @@ const SignIn = ({ onSignIn }) => {
     })
       .then(response => {
         if (response.status === 200)
-          confirmUser(response.json(), history);
+          confirmUser(response.json());
         else if (response.status === 401)
           alert('Incorrent email or password!');
```
Comment (zigsas) [inline frontend/src/pages/SignIn.js:28]: There's a typo in word `Incorrect`

Comment (tozaicevas) [reply frontend/src/pages/SignIn.js:28]: Good, we need to leave the place cleaner than it was

**Comment by tozaicevas** [comment]
Closing this, https://github.com/UndeadRat22/epicenter-tp-learning-calendar/pull/13 has these changes.

**Pull request closed without merging**